types/react-image-fallback/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for react-image-fallback 8.0
+// Project: https://github.com/socialtables/react-image-fallback
+// Definitions by: Ben Smith <https://github.com/8enSmith>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as React from "react";
+
+export class ReactImageFallback extends React.Component<
+    ReactImageFallbackProps &
+    React.DetailedHTMLProps<
+    React.ImgHTMLAttributes<HTMLImageElement>,
+    HTMLImageElement
+    >,
+    any
+    > { }
+
+export interface ReactImageFallbackProps {
+    src: string;
+    fallbackImage: string | React.ReactElement<any> | any[];
```

Comment:
Can you please narrow `any[]` here?

Reply:
As per the [documentation](https://github.com/socialtables/react-image-fallback#fallbackimage), fallbackImage should be a string or a React element. As it's not specific with regards to what type of ReactElement it should be I've used 'any'. Suggestions?

Reply:
Ah yes, I'll get this fixed ASAP.

Reply:
I've pushed a change with more specific typings as per the package [documentation](https://github.com/socialtables/react-image-fallback#fallbackimage).